unit-tests/run-unit-tests.hs

```diff
@@ -46,6 +46,42 @@ test_reload_modified = TestCase "reload_modified" [mod_file] $ do
                         setTopLevelModules [mod_name]
                         interpret "f" (as :: Int -> Int)
 
+test_reload_modified :: TestCase
+test_reload_modified = TestCase "reload_modified" [mod_file_a, mod_file_b] $ do
+                             liftIO $ writeFile mod_file_a mod_a_v1
+                             loadModules [mod_file_a]
+                             setTopLevelModules [mod_name_a]
+                             f_v1 <- interpret "fin" (as :: String -> String)
+                             let a = (f_v1 "foo")
+                             --
+                             liftIO $ writeFile mod_file_a mod_a_v2
+                             liftIO $ writeFile mod_file_b mod_b
+                             set [searchPath := ["."]]
```

Comment:
I'm not quite sure why this is necessary, will need to spend some time debugging. It breaks if you move it to the top, which isn't good!